src/decaylanguage/dec/dec.py

```diff
@@ -692,11 +692,12 @@ def _decay_mode_details(
     def print_decay_modes(
         self,
         mother: str,
-        pdg_name: Optional[bool] = False,
-        print_model: Optional[bool] = True,
-        display_photos_keyword: Optional[bool] = True,
-        ascending: Optional[bool] = False,
-        normalize: Optional[bool] = True,
+        pdg_name: bool = False,
+        print_model: bool = True,
+        display_photos_keyword: bool = True,
+        ascending: bool = False,
+        normalize: bool = False,
```

Comment:
FYI @yipengsun the normalisation is now set to False so that by default no values are "massaged". I think this is potentially clearer and more the expected behaviour,